### PR TITLE
Access to change classes

### PR DESCRIPTION
**Pull request by skylei**
This method is more popular.

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -263,7 +263,7 @@ Connection pools are configured via the `setConnectionPool()` method:
 
 [source,php]
 ----
-$connectionPool = '\Elasticsearch\ConnectionPool\StaticNoPingConnectionPool';
+$connectionPool = \Elasticsearch\ConnectionPool\StaticNoPingConnectionPool::class;;
```
Comment (mhujer) [inline docs/configuration.asciidoc]: Typo (duplicated semicolon)

```suggestion
$connectionPool = \Elasticsearch\ConnectionPool\StaticNoPingConnectionPool::class;
```

Comment (skylei) [reply docs/configuration.asciidoc]: It has been modified. Thank you！

**Comment by polyfractal** [comment]
Thanks!  Would you mind [Signing the CLA](https://www.elastic.co/contributor-agreement/) so that we can merge this?

(Sorry for the long delay!)

**Comment by ezimuel** [comment]
@mhujer the CLA is still missing, you can sign it [here](https://www.elastic.co/contributor-agreement).
I'm closing this PR until you will sign it, thanks.

**Comment by mhujer** [comment]
@ezimuel you probably meant @skylei, not me, right?

**Comment by ezimuel** [comment]
@mhujer yes, sorry. @skylei the CLA is still missing, I closed the PR waiting for that. Thanks.

**Pull request closed without merging**